privaterelay/views.py

```diff
@@ -254,6 +254,15 @@ def _handle_fxa_profile_change(authentic_jwt, social_account, event_key):
         return HttpResponse('202 Accepted', status=202)
 
     extra_data = resp.json()
+
+    if resp.status_code != 200:
+        sentry_sdk.capture_message(
+            'FXA Profile fetch error.',
+            level="error",
+            stack=True
```

Comment:
FYI this `stack=True` does not actually return the stack because this is a `capture_message`. If you want the stack it's best to use `logger.error` to capture the message or let the `extra_data['email']` throw an exception and capture the exception using `sentry_sdk.capture_exception` which sends stack trace by default.

Reply:
Oh good point. Updated to catch and capture the exception and return the 202 to FXA to stop re-trying.